docker/workspace-build/Dockerfile

```diff
@@ -38,6 +40,9 @@ RUN apt-get -y update \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
+# For the Demo UI
+RUN source /opt/ros/${ROS_DISTRO}/setup.bash
+
```

Comment:
This has no effect beyond this layer. Is this "cruft" at this point, or was there an intention behind keeping this around?